packages/npm-dependencies-validation/src/depIssuesFinder.ts

```diff
@@ -42,6 +40,21 @@ export async function findDependencyIssues(
     devDepsCommandConfig
   );
   /* istanbul ignore next  -- inconsistent behavior under npm6-8 so not all branches are reachable here */
```

Comment:
you can remove this coverage comment as well now

Reply:
Done